manifests/telegraf/deployment.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  creationTimestamp: null
```

Comment:
@durandom these should be left in the files, I think openshift or argocd schema validation depends on it.